frontend/invites/inviteInstitutionController.js

```diff
@@ -159,6 +162,14 @@
             }
         }
 
+        function isSuperUser() {
```

Comment:
This function should be a User function.

Reply:
Done